recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "gensim" %}
-{% set version = "3.5.0" %}
-{% set sha256 = "78ed9b6ac35f104542f3bee0386d71ddf9432d74c153065d2ea9f6baf10e5b49" %}
+{% set version = "3.7.0" %}
+{% set sha256 = "0edd7ebc742a15bf2b2fefa91d3e5c57f186e821314a1e6a793b988326388c50" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/conda-forge/gensim-feedstock/files/2775378/gensim-3.7.0.tar.gz
```

Comment:
@menshikh-iv this URL looks wrong

Reply:
yeah, this is temporary only for testing